src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.html

```diff
@@ -60,6 +60,7 @@
         </cd-dashboard-area-chart>
         <cd-dashboard-area-chart chartTitle="Latency"
                                  dataUnits="ms"
```

Comment:
@Pegonzal , I was thinking to go this way as well but I saw in the dashboard-area-chart component we expect the value to be in `ms` in the `convertToChartDataUnits` method that converts the incoming value to the corresponding data unit i.e ms/s/ks/Ms etc. Hence we need to send value in `ms` to the dashboard chart component.

Reply:
yes I would agree here too. We are adding one more operation in query which is kind of we can avoid.
We may improve our helper function here surely.

Reply:
@afreen23 but everywhere else in dashboard we are showing/passing the Latency values in ms. If you check the Cluster Utilization Chart on the Landing page, Latency is shown in ms only and the metric used returns the data in ms. In grafana dashboards as well we have the latency in ms by default. In order to be consistent everywhere across the code I think its better to pass the values in ms here instead of s.

Reply:
Okay thanks for clarifying, This should be good then.

Reply:
Fine for me as well. I think maybe we should look to improve this charts behaviour to avoid having to deal with data units transformations.
That's, however, out of the scope of this PR so I'm fine with it as well